integration-tests/src/test/java/oracle/kubernetes/operator/ItJrfPvWlst.java

```diff
@@ -14,133 +14,167 @@
+    assertDoesNotThrow(() -> DbUtils.deleteRcuPod(getResultDir()),
+        "Failed: drop RCU schema");
+    assertDoesNotThrow(() -> DbUtils.deleteDbPod(getResultDir()),
+        "Failed: stop DB");
   }
   
   /**
-  * This method will run once after all test methods are finished. It Releases k8s cluster lease,
-  * archives result, pv directories.
-  *
-  * @throws Exception - if any error occurs
-  */
+   * Releases k8s cluster lease, archives result, pv directories.
+   */
   @AfterAll
-  public static void staticUnPrepare() throws Exception {
-    tearDown(new Object() {
-    }.getClass().getEnclosingClass().getSimpleName(), namespaceList.toString());
-
-    LoggerHelper.getLocal().log(Level.INFO,"SUCCESS");
+  public static void staticUnPrepare() {
+    assertDoesNotThrow(() -> tearDown(new Object() {
+        }.getClass().getEnclosingClass().getSimpleName(), namespaceList.toString()),
+         "tearDown failed");
   }
- 
+
+  /**
+   * Create and deploy a JRF domain. Verify the domain is created successfully
+   */
   @Test
   public void testJrfDomainOnPvUsingWlst() throws Exception {
     if (QUICKTEST) {
       String testMethodName = new Object() {
       }.getClass().getEnclosingMethod().getName();
       logTestBegin(testMethodName);
-      LoggerHelper.getLocal().log(Level.INFO,
-          "Creating Operator & waiting for the script to complete execution");
       
-      JrfDomain jrfdomain = null;
       boolean testCompletedSuccessfully = false;
 
       try {
         // create JRF domain
         Map<String, Object> domainMap = createDomainMap(getNewSuffixCount(), testClassName);
         domainMap.put("namespace", domainNS);
         domainMap.put("initialManagedServerReplicas", new Integer("2"));
-        domainMap.put("image", "container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4");
         domainMap.put("clusterName", "infra-cluster");
         domainMap.put("managedServerNameBase", "infraserver");
+        domainMap.put("domainHomeSourceType", "PersistentVolume");
         domainMap.put("rcuSchemaPrefix", "jrfdomain");
-        domainMap.put("rcuDatabaseURL", "oracle-db.default.svc.cluster.local:1521/devpdb.k8s");
+        LoggerHelper.getLocal().log(Level.INFO, "DEBUG " + testClassName + "domain: dbUrl: " 
+            + dbUrl);
+        domainMap.put("rcuDatabaseURL", dbUrl);
         domainUid = (String) domainMap.get("domainUID");
         LoggerHelper.getLocal().log(Level.INFO,
             "Creating and verifying the domain creation with domainUid: " + domainUid);
 
-        jrfdomain = new JrfDomain(domainMap);
-        jrfdomain.verifyDomainCreated(80);
+        jrfdomain = assertDoesNotThrow(() -> new JrfDomain(domainMap), 
+            "Failed: JRF domain creation");
+        LoggerHelper.getLocal().log(Level.INFO, "JRF domain is created for {0}", 
+            testClassName);
+        assertDoesNotThrow(() -> jrfdomain.verifyDomainCreated(60), 
```

Comment:
magic number

Reply:
Can you please review the explanation for the same question.

Reply:
Can you please review the explanation for the same question.